app/views/navigation_rules/edit.html.erb

```diff
@@ -18,7 +18,7 @@
 <% content_for :title, @step_by_step_page.title %>
 <% content_for :context, "Sidebar settings" %>
 
-<%= render "shared/steps/form_notice", message: notice, status: "success" %>
+<%= render "shared/steps/form_notice", message: notice, status: nil %>
```

Comment:
What is this (and a similar change in `app/views/step_by_step_pages/index.html.erb`) about?

Reply:
I have restored these to their original state now since these changes are not necessary for this piece of work.